src/vst/ivstunits.rs

```diff
@@ -1,94 +1,102 @@
-use crate::base::{tchar, tresult, IBStream};
-use crate::vst::{CString, String128};
-use vst3_com::com_interface;
+use crate::base::tresult;
 use vst3_com::interfaces::iunknown::IUnknown;
-//todo: ivstunit constants
+use vst3_com::{c_void, com_interface};
+
+pub const kRootUnitId: i32 = 0;
+pub const kNoParentUnitId: i32 = -1;
+
+pub const kNoProgramListId: i32 = -1;
 
 #[repr(C)]
 #[derive(Copy, Clone)]
 pub struct UnitInfo {
     pub id: i32,
     pub parent_unit_id: i32,
-    pub name: String128,
+    pub name: [u16; 128],
```

Comment:
We should probably have a similar wrapper name around this to indicate that it's a widestring, but unsigned.

Reply:
That's fine, I can revert to String128.

Reply:
Reverted